src/main/xerus/monstercat/api/response/Track.kt

```diff
@@ -21,6 +21,8 @@ open class Track: MusicItem() {
 	var artists: List<ArtistRel> = emptyList()
 	@Key
 	var bpm: Double? = null
+	@Key
+	var licensable: Boolean = false
```

Comment:
If you add a new Key, you need to update the Cache version, otherwise it won't be available for the already cached items.

Reply:
09c86cc85a58a920fd3813c044cd73f03e7791aa